README.md

```diff
@@ -10,51 +10,142 @@
 
 [Contentful](https://www.contentful.com) provides a content infrastructure for digital teams to power content in websites, apps, and devices. Unlike a CMS, Contentful was built to integrate with the modern software stack. It offers a central hub for structured content, powerful management and delivery APIs, and a customizable web app that enable developers and content creators to ship digital products faster.
 
-This Command Line Tool (CLI) helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
+This library helps you to import files generated by [contentful-export](https://github.com/contentful/contentful-export) to a destination space.
 
-## Installation
+## :exclamation: Usage as CLI
+> We moved the CLI version of this tool into our [Contentful CLI](https://github.com/contentful/contentful-cli). This allows our users to use and install only one single CLI tool to get the full Contentful experience.
+>
+> Please have a look at the [Contentful CLI import command documentation](https://github.com/contentful/contentful-cli/tree/master/docs/space/import) to learn more about how to use this as command line tool.
 
-We recommend you install the CLI via npm:
+## :cloud: Installation
 
 ```bash
-npm install -g contentful-import
+npm install contentful-import
 ```
 
-## Usage and examples
+## :hand: Usage
 
-### Example
+```javascript
+const contentfulImport = require('contentful-import')
 
-#### Import an environment
+const options = {
+  content: {entries:..., contentTypes:..., locales:...},
```

Comment:
maybe add `const contentfulImport = require('contentful-import')` to stay consistent with other snippets